docs/riskAuthentication.md

```diff
@@ -4,16 +4,36 @@ Extra steps may be required to set up the widget when using adapters with risk-b
 
 **Table of Contents**
 - [ID DataWeb Integration Kit](#id-dataweb-integration-kit)
+- [ThreatMetrix Integration Kit](#threatmetrix-integration-kit)
 
 ## ID DataWeb Integration Kit
 
-### Dependencies requirement
+### System requirements and dependencies
 
 * ID DataWeb Integration Kit 1.1 or later
 
 ### Setup
 
 If you have configured the adapter so that the `Device Profiling Method` is set to `Captured by this adapter`, no extra setup is required as the widget will be able to run the device profiling script with information provided by the adapter. However if you have chosen `Captured by a previous adapter` you will need to place the device profiling script that came with the kit (`id_dataweb_device_profiling.js`) to where your application will be hosted and attach the following script in the index.html that you have created. 
 ```html
-<script language="javascript" src="id_dataweb_device_profiling.js"></script>
+<script type="text/javascript" src="id_dataweb_device_profiling.js"></script>
+```
+
+## ThreatMetrix Integration Kit
+
+### System requirements and dependencies
+
+* ThreatMetrix Integration Kit 1.1 or later
+
+### Setup
+
+If you have configured the adapter so that the `Device Profiling Method` is set to `Captured by this adapter` and `Device Profiling Script Source` is set to `ThreatMetrix Web` then no extra setup is required. However if `Device Profiling Script Source` is set to `ThreatMetrix SDK`, you will need to make sure `deviceProfileScript` is included when `PfAuthnWidget` is initialized.
```

Comment:
Looks better but I think you need a comma after "However". It should probably read. "However, if `Device Profiling Script Source` is set to `ThreatMetrix SDK`, you will need to make sure `deviceProfileScript` is included when `PfAuthnWidget` is initialized."

Let's get Mike to review this README before submitting.

Reply:
I'll wait until Mike is back to continue on this then, thanks!